src/styles/components/button.scss

```diff
@@ -13,3 +13,19 @@
     box-shadow: none !important;
   }
 }
+
+.govuk-button-secondary {
+  background-color: white !important;
+  &:hover {
+    background-color: repairs-hub-colour('housing-2') !important;
+  }
+  &:focus {
+    opacity: 0.9 !important;
+    background-color: repairs-hub-colour('housing-2') !important;
+    border-color: repairs-hub-colour('housing-2') !important;
+  }
+  &:focus,
+  &:hover {
+    box-shadow: none !important;
+  }
+}
```

Comment:
Seems some duplicate stuff here as we see in `govuk-button` above. Can we drop some of these css declarations and only keep what is necessary? I believe the entire class will have to have `govuk-button govuk-button-secondary` .

Reply:
Where are we using this as well? Think I can't see where it's being used in the changes